tests/test_intf_mac.py

```diff
@@ -0,0 +1,258 @@
+from swsscommon import swsscommon
+
+import time
+import json
+
+class TestRouterInterfaceMac(object):
+    def setup_db(self, dvs):
+        self.pdb = swsscommon.DBConnector(swsscommon.APPL_DB, dvs.redis_sock, 0)
+        self.adb = swsscommon.DBConnector(swsscommon.ASIC_DB, dvs.redis_sock, 0)
+        self.cdb = swsscommon.DBConnector(swsscommon.CONFIG_DB, dvs.redis_sock, 0)
+        self.cntdb = swsscommon.DBConnector(swsscommon.COUNTERS_DB, dvs.redis_sock, 0)
+
+    def add_ip_address(self, interface, ip):
+        tbl = swsscommon.Table(self.cdb, "INTERFACE")
+        fvs = swsscommon.FieldValuePairs([("NULL", "NULL")])
+        tbl.set(interface + "|" + ip, fvs)
+        time.sleep(2) # IPv6 netlink message needs longer time
+
+    def remove_ip_address(self, interface, ip):
```

Comment:
these functions are already defined in `conftest.py` file. you could re-use them.

Reply:
The vstest has been updated according to your comments, please review again, thanks.